client/components/status/status.js

```diff
@@ -537,10 +537,9 @@ export default class Status extends React.Component {
               {links && links.map(link => {
                 if (shouldLinkBeShown(link, isAuthenticated)) {
                   return (
-                    <div className="links row">
+                    <div className="links row" key={link}>
```

Comment:
This is not perfect solution. You will get ``` Encountered two children with the same key, `[object Object]``` warning for multiple objects.

Reply:
Take reference from this conversation. https://github.com/openwisp/openwisp-wifi-login-pages/pull/181#discussion_r592025358

Reply:
@niteshsinha17  
It would nice to hear some suggestions on facts,  on grounds of which i conducted this approach :-
* For multiple render() , key for same element must be same ( just to not create that element multiple times ) ?
* Also  key must present on outer element otherwise the react will take outer element as new element every time unless you provide a key to it to make react understand that okay this is same last element and react wont recreate it in dom ?

also
I tried this approach and i did not get any such error , am i missing something to reproduce the error you mentioned ?

Reply:
1. We can not have the same key for different elements wheater they are the same or different. 

2. Yes, key must be present on the outer element. This was the only reason for which we were getting key warning. 

3.In this situation, you can use index or url as the key. 
I also understand we should not use the index as the key but this is true when the list gets changed every time. Here we are showing a button on the status component with the help of a list that may not change again and again. 

4. Also you can't reproduce that warning here. But see the image below. It will help you to understand.

![Screenshot (164)](https://user-images.githubusercontent.com/42765292/111815963-4675d780-8902-11eb-9300-707145604f03.png)

Reply:
@niteshsinha17  thank you for the explanation.
for the last point , i think this is the case when we would have same `links` which i dont think we would? , so for different `link` we would have different key .

Also, if i am able to  sum up correctly ,  only thing left is to assign `key={link.url}` at  outer div ?

Reply:
Yes, you are right